include/z64save.h

```diff
@@ -388,13 +388,13 @@ typedef enum {
 #define EVENTCHKINF_0B 0x0B
 #define EVENTCHKINF_0C 0x0C
 #define EVENTCHKINF_0F 0x0F
-#define EVENTCHKINF_10 0x10
+#define EVENTCHKINF_TALKED_TO_MALON_FIRST_TIME 0x10
 #define EVENTCHKINF_11 0x11
-#define EVENTCHKINF_12 0x12
+#define EVENTCHKINF_RECEIVED_WEIRD_EGG 0x12
 #define EVENTCHKINF_TALON_WOKEN_IN_CASTLE 0x13
 #define EVENTCHKINF_TALON_RETURNED_FROM_CASTLE 0x14
-#define EVENTCHKINF_15 0x15
-#define EVENTCHKINF_16 0x16
+#define EVENTCHKINF_TALKED_TO_CHILD_MALON_AT_RANCH 0x15
+#define EVENTCHKINF_CAN_OBTAIN_EPONA_SONG 0x16
 #define EVENTCHKINF_EPONA_OBTAINED 0x18
```

Comment:
EPONA_SONG -> EPONAS_SONG
and maybe instead of "can obtain", "can learn"? cause you have to pull your ocarina out and go through the whole song learning sequence, then you obtain it later